static/scss/profile.scss

```diff
@@ -32,6 +32,9 @@
     flex-direction: row;
     color: $lightgrey !important;
     text-transform: uppercase;
+    position: absolute;
+    right: 0;
+    padding-right: 41px;
```

Comment:
@amir-qayyum-khan do you have a reason to keep them around? if not let's delete.